messageBuilder.js

```diff
@@ -0,0 +1,65 @@
+const messages = {
+  prTitle: 'PR Title',
+  prBody: 'PR Description',
+  spaceBetweenHashtagAndDigit: 'PR contains a space between the hashtag and digit.',
+  noIssueReference: 'Issue Reference (`#<issue-number>`) missing.',
+  missingGithubKeyword: 'GitHub Keyword missing: Refer [here](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) for a list of accepted keywords.',
+};
+
+function isKeywordCheckerEnabled() {
+  return process.env.ENABLE_KEYWORD_CHECKER !== undefined &&
+    process.env.ENABLE_KEYWORD_CHECKER.toLowerCase() === 'true';
+}
+
+/*
+* Formats message as a GFMD level two unordered list item
+*/
+function getFormattedMessageLevelTwoUnordered(message) {
+  return `   * ${message}\n`;
+}
+
+/*
+* Formats message as a GFMD level two unordered list item
+*/
+function getFormattedMessageLevelOneOrdered(message) {
+  return `1. ${message}\n`;
+}
+
+function buildTitleFeedback(violations) {
+  let message = '';
+  if (violations === undefined) return message;
```

Comment:
If you are going to use `buildTitleFeedback` in other parts and you forgot to pass in violations, it will return blank without any error. Hence, it's quite hard for us to debug if that happens (i.e. is it expected behaviour)?